extensions/wikia/ArticleExporter/ArticleExporterHooks.class.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Wikia\Rabbit\ConnectionBase;
+
+/**
+ * @author Lore team
+ *
+ * Send plaintext versions of articles to RabbitMQ as they are edited on Fandom
+ */
+
+class ArticleExporterHooks {
+    protected static $rabbitConnection;
+
+    public static function onArticleSaveComplete( WikiPage $page ) {
+        global $wgCityId;
+        global $wgArticleExporterExchange;
+
+        $message = [
+            'wikiId' => $wgCityId,
+            'pageIds' => [ strval($page->getId()) ],
+        ];
+
+        self::getRabbitConnection()->publish( $wgArticleExporterExchange['routing'], $message );
```

Comment:
Are there any namespaces we should ignore?

Reply:
Could you clarify this comment? Thanks

Reply:
I _think_ we probably only care about namespace 0 but I could be wrong. This [page](https://www.mediawiki.org/wiki/Manual:Namespace) has the list of all of the namespaces.

For now, we can probably safely ignore the category, talk, file, and user pages?

Reply:
It might also be ok to ignore them downstream but if we're concerned about message volume we may want to consider filtering here. I don't have any insight into what kind of volume those pages generate. Generally, I would guess it's pretty low except for maybe the talk pages on some communities?

Reply:
I agree. I think we should do both -- ignore non `0` namespaces and include the namespace for each page. That might cause us to reconsider the payload to something like:
```json
{
  "wikiId": ...,
  "pages": [{"pageId": ..., "namespace": ...}]
}
```

If we choose to do that, I can make the corresponding change in Wikia/pandora#6427 to take in the new structure and ignore non-`0` namespaces. What do you think @drsnyder @jamieshepherd ?

Reply:
That's a good callout @drsnyder - updated in ef76cd7

Reply:
Sounds good.